draft-linker-adem-wg-adem-core.md

```diff
@@ -185,6 +196,9 @@ To resolve an EI, it is first interpreted as an implicit or explicit set of addr
 If `address` is an IP address, the set contains this address only.
 If it is an IP address prefix, it contains all addresses matching that prefix.
 If it is a domain name, it contains any IP address this domain name can be resolved to.
+
+TODO(djackson) - I'm not sure about "any address this domain can resolve to" - AFAIK DNS Servers can and will respond with a rotating list of IP addresses for a domain name. I.e. you get a single A record each time, but it changes every 5 minutes.
```

Comment:
I start to get the feeling more and more that we maybe shouldn't protected domain names at all.